src/modules/Snapper.rb

```diff
@@ -23,14 +23,119 @@
 # Summary:	Snapper settings, input and output functions
 # Authors:	Jiri Suchomel <jsuchome@suse.cz>
 #
-# $Id$
-#
 # Representation of the configuration of snapper.
 # Input and output routines.
+
 require "yast"
 
 module Yast
+
+
+  class Tree
```

Comment:
I made Tree an inner class of SnapperClass now. It's also ugly since SnapperClass is only 	an auxiliary class later assigned to Snapper but still has to be used in the unit-tests.